src/main/java/com/dev/cinema/controllers/CinemaHallController.java

```diff
@@ -0,0 +1,38 @@
+package com.dev.cinema.controllers;
+
+import com.dev.cinema.controllers.mapper.CinemaHallMapper;
+import com.dev.cinema.model.dto.CinemaHallRequestDto;
+import com.dev.cinema.model.dto.CinemaHallResponseDto;
+import com.dev.cinema.service.CinemaHallService;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema-hall")
```

Comment:
```suggestion
@RequestMapping("/cinema-halls")
```
let's change this to plural everywhere